web/static/js/actions/respondents.js

```diff
@@ -37,7 +38,8 @@ export const fetchRespondents = (
             response.result,
             filter,
             sortBy,
-            sortAsc
+            sortAsc,
+            response.respondentsFields
```

Comment:
I understand that this respondentFields will be used as selected fields only in certain situations (empty selectedFields for example). But from this code that is not 100% clear.